airflow/kubernetes/pod_generator_deprecated.py

```diff
@@ -178,6 +178,11 @@ def __init__(
 
         self.container.command = cmds or []
         self.container.args = args or []
+        if not image_pull_policy:
+            if image and image.endswith("latest"):
```

Comment:
This probably also needs to handle images without tags (e.g. `debian`). Maybe:

```suggestion
            if image and (image.endswith("latest") or ":" in image):
```

Reply:
Ah yeah. but it should be ` or ":" not in image:`

Reply:
Oh, yeah, oops 🤦‍♂️